modules/core/client/controllers/groupController.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+angular.module('core').controller('GroupCtrl', function ($scope, $log) {
```

Comment:
I know it is the log output being used but someone might not know that on the team